src/haxelib/client/Main.hx

```diff
@@ -1037,17 +1048,17 @@ class Main {
 		}
 		else
 		{
-			var inf = try site.infos(p) catch( e : Dynamic ) { Sys.println(e); return; };
-			p = inf.name;
```

Comment:
You probably meant `rep` instead of `pdir`, but yeah, that might work! Please factor that code into a separate function tho :)